cmd/argocd/commands/applicationset_test.go

```diff
@@ -185,7 +185,7 @@ SyncPolicy:         Automated
 			printAppSetSummaryTable(tt.appSet)
 			w.Close()
 
-			out, err := ioutil.ReadAll(r)
+			out, err := io.ReadAll(r)
```

Comment:
```suggestion
			out, err := os.ReadAll(r)
```

Reply:
![image](https://user-images.githubusercontent.com/41536271/219876854-b833673f-7a1c-41a0-a3ed-b59944285f5c.png)

I was change `io.ReadAll` to `os.ReadAll` and remove import `io`.  
But, `os.ReadAll` is not exists.  

so, git reset and force push before commit